tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs

```diff
@@ -49,9 +70,11 @@ public async Task RunAsync(TimeSpan waitPeriodAfterEachUpdate, CancellationToken
             }
         }
 
-        private async Task<Dictionary<BuildDefinitionId, DateTime>> ImportVstsBuildsDataAsync(BuildManagement buildManagement, string branch, HashSet<BuildDefinitionId> buildDefinitionIds)
+        async Task<Dictionary<BuildDefinitionId, DateTime>> ImportVstsBuildsDataAsync(BuildManagement buildManagement, string branch, HashSet<BuildDefinitionId> buildDefinitionIds)
         {
-            Dictionary<BuildDefinitionId, DateTime> lastUpdatePerDefinition = this.lastUpdatePerBranchPerDefinition.GetIfExists(branch);
+            Console.WriteLine($"Import VSTS builds from branch [{branch}]");
+            Dictionary<BuildDefinitionId, DateTime> lastUpdatePerDefinition = this.buildLastUpdatePerBranchPerDefinition.GetIfExists(branch);
```

Comment:
nit: it could be more obvious that GetIfExists returns a datetime

Reply:
no, each branch has multiple build definition; and each definition should have its own last update timestamp.